operator/Makefile

```diff
@@ -353,42 +353,42 @@ deploy: check-ci-setup manifests kustomize ## Deploy operator image to the K8s c
 		cd config/local-deploy-versioned && \
 		$(KUSTOMIZE) create --resources ../default && \
 		$(KUSTOMIZE) edit set image quay.io/stackrox-io/stackrox-operator=$(IMG)
-	$(KUSTOMIZE) build config/local-deploy-versioned | kubectl create -f -
+	$(KUSTOMIZE) build config/local-deploy-versioned | $(PROJECT_DIR)/hack/retry-kubectl.sh apply -f -
```

Comment:
Changed `create` to `apply`, I suppose it's intentional

Reply:
Yes, `create` is not idempotent so not safe for retrying.